katello/katello-restore

```diff
@@ -46,6 +46,7 @@ def restore_databases
     `service postgresql start`
     `runuser - postgres -c "dropdb foreman"`
     `runuser - postgres -c "dropdb candlepin"`
+    `runuser - postgres -c "psql -f #{DIR}/pg_globals.dump postgres"`
```

Comment:
I'm getting some errors here that can be ignored. Could you add ` 2>/dev/null` to the end of this command? Thanks @jcpunk !